supporting-blog-content/introducing-retrievers/retrievers_intro_notebook.ipynb

```diff
@@ -0,0 +1,1047 @@
+   ],
+   "metadata": {
+    "id": "Mr2fj6zHfDIx"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "rj = response.json()\n",
+    "cloud_id = rj[\"cloud_id\"]\n",
+    "cloud_username = rj[\"credentials\"][\"username\"]\n",
+    "cloud_password = rj[\"credentials\"][\"password\"]"
+   ],
+   "metadata": {
+    "id": "NohlsUagch98"
+   },
+   "execution_count": 7,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Create Elasticsearch connection"
+   ],
+   "metadata": {
+    "id": "Tq_UuqxPsxlJ"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "es = Elasticsearch(cloud_id=cloud_id, basic_auth=(cloud_username, cloud_password))\n",
+    "\n",
+    "# Wait for project to be created and available\n",
+    "while True:\n",
+    "    sleep(0.5)\n",
+    "    try:\n",
+    "        if es.ping():\n",
+    "            print(\"project created\")\n",
+    "            break\n",
+    "    except ConnectionError:\n",
+    "        pass"
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "AQepIFokswoR",
+    "outputId": "5230db4e-e474-4f8d-ad93-2c2fa4fc21f5"
+   },
+   "execution_count": 8,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "project created\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Deploy Elser and e5\n",
+    "The two blocks below will deploy the embedding models and auto-scale ML capacity"
+   ],
+   "metadata": {
+    "id": "2qeuIvZhFjyH"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Deploy and start ELSER"
+   ],
+   "metadata": {
+    "id": "EbKE5-11F-HA"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "try:\n",
+    "    resp = es.inference.put_model(\n",
+    "        task_type=\"sparse_embedding\",\n",
+    "        inference_id=\"my-elser-model\",\n",
+    "        body={\n",
+    "            \"service\": \"elser\",\n",
+    "            \"service_settings\": {\"num_allocations\": 64, \"num_threads\": 1},\n",
+    "        },\n",
+    "    )\n",
+    "except ConnectionTimeout:\n",
+    "    pass"
+   ],
+   "metadata": {
+    "id": "PLvy4pY6FnPF"
+   },
+   "execution_count": 9,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Deploy and start e5-small"
+   ],
+   "metadata": {
+    "id": "4DVmAYW8GBAo"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "try:\n",
+    "    resp = es.inference.put_model(\n",
+    "        task_type=\"text_embedding\",\n",
+    "        inference_id=\"my-e5-model\",\n",
+    "        body={\n",
+    "            \"service\": \"elasticsearch\",\n",
+    "            \"service_settings\": {\n",
+    "                \"num_allocations\": 8,\n",
+    "                \"num_threads\": 1,\n",
+    "                \"model_id\": \".multilingual-e5-small_linux-x86_64\",\n",
```

Comment:
should be able to use `.multilingual-e5-small` and inference will pick `_linux-x86_64` if supported.